scripts/run-ci-tests

```diff
@@ -4,11 +4,10 @@
 # Supports both acceptance tests and migration tests
 # Dynamically detects which services have sweeper implementations by grepping for AddTestSweepers
 
-# ./internal/services/api_token
-# 'api_token' requires token-based auth, and our current CI assumes key-based auth.
-# These tests otherwise pass.
-# TODO: support multiple auth schemes in CI for acceptance tests
-
+# ./internal/services/account
+# account creation can only be done by a tenant admin and the current CI does 
+# have that support.
+# TODO: support tenant admin auth for acceptance tests
```

Comment:
This part has been mentioned already but the account tests are currently set to skip so they don't need to be removed from this list